packages/Webkul/Installer/src/Console/Commands/Installer.php

```diff
@@ -427,6 +448,7 @@ protected function allowedChoice(string $question, array $choices)
         $selectedValues = multiselect(
             label: $question,
             options: array_values($choices),
+            required: true,
```

Comment:
@ghermans ,

If no options are selected in the "Allowed Currencies" and "Allowed Locales" sections, the default currency and locales will be automatically applied. Therefore, it is not necessary to make this a required step. If there are specific reasons for making this mandatory, please provide an explanation.